<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>educational_program</class>
 <widget class="QWidget" name="educational_program">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1061</width>
     <height>351</height>
    </rect>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>931</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>431</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Образовательная программа</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>140</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>170</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <action name="action">
   <property name="text">
    <string>Дисциплины</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Дисциплины учебного плана</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Компетенции</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Компетенции дисциплины</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Образовательный стандарт</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Образовательная программа</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Результат</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
